src/Interpreters/ProcessorsProfileLog.cpp

```diff
@@ -26,12 +26,18 @@ NamesAndTypesList ProcessorProfileLogElement::getNamesAndTypes()
 
         {"id", std::make_shared<DataTypeUInt64>()},
         {"parent_ids", std::make_shared<DataTypeArray>(std::make_shared<DataTypeUInt64>())},
+        {"plan_step", std::make_shared<DataTypeUInt64>()},
+        {"plan_group", std::make_shared<DataTypeUInt64>()},
```

Comment:
Hi @amosbird, may I know the purpose of adding plan_step into the log? Can we use it to get additional information about the processor, e.g., the table/column processed by it? Thank you!

Reply:
It's used to identify processors as a group. See https://github.com/ClickHouse/ClickHouse/blob/master/src/QueryPipeline/printPipeline.h#L69 . Though it's not perfect.